plugins/modules/api_modify.py

```diff
@@ -87,6 +87,18 @@
         - interface sstp-server server
         - interface vlan
         - interface vrrp
+        - interface wifiwave2
+        - interface wifiwave2 aaa
+        - interface wifiwave2 access-list
+        - interface wifiwave2 cap
+        - interface wifiwave2 capsman
+        - interface wifiwave2 channel
+        - interface wifiwave2 configuration
+        - interface wifiwave2 datapath
+        - interface wifiwave2 interworking
+        - interface wifiwave2 provisioning
+        - interface wifiwave2 security
+        - interface wifiwave2 steering
```

Comment:
About tests... I'm sorry I didn't add any, but I can't think about a good one. I'm not a big fan of tests which just compare requests and responses (that would be job of DSL -> request converter and its tests).

Probably a good integration one is starting a real RouterOS (probably X64) in QEMU...

Reply:
Tests aren't really needed. It would be great to have proper tests for this (and other paths), but these require starting a real Router OS as you mentioned, which isn't so trivial.

If someone wants to work on adding something like that to CI, I'll happily help, but I don't have the resources to do this myself :)